src/components/content-tab-import-export.tsx

```diff
@@ -93,14 +99,21 @@ const InitialImportButton = ( {
 	disabled?: boolean;
 } ) =>
 	isInitial ? (
-		<Button
-			variant="icon"
-			className="w-full [&>div.border-zinc-300]:hover:border-a8c-blueberry"
-			onClick={ openFileSelector }
-			disabled={ disabled }
-		>
-			{ children }
-		</Button>
+		<Tooltip className={ 'w-full' } text={ siteSyncingMessage() } disabled={ ! disabled }>
```

Comment:
We need to add a similar condition as `isSyncing`. If not the tooltip in the drop down area is incorrect.

```suggestion
		<Tooltip
			className="w-full"
			text={ isSyncing ? siteSyncingMessage() : siteExportingMessage }
			disabled={ ! disabled }
		>
```

Steps to reproduce:
1. Access the import/export tab
2. Click on export entire site
3. Move the mouse hover the drop down area
4. Observe the tooltip with the Sync message , instead of seeing a message about the site being exported.


https://github.com/user-attachments/assets/88a95fda-d877-4e45-b1fa-10811a8e1226

Reply:
That's a great catch! Thanks, Antonio! I will address this issue tomorrow morning.

Reply:
I have addressed this issue. 🙂